### ISSUE TITLE
Missing License?

### ISSUE THREAD

**Issue by ax3l** [psc-code/psc#74]
Hi Kai,

I was wondering which open-source license PSC uses but did not find a `LICENSE` file in the root of the repository or in individual file headers.

Is this missing?

Thanks for your help,
Axel

**Comment by germasch** [comment]
Yeah, I haven't really made a final decision on that. I guess you can assume GPL for now, though I may go with something less restrictive eventually. There's a lot of work that needs to be finished before this is anywhere near ready for general consumption...

**Comment by ax3l** [comment]
Thanks for the quick feedback!

Yes, no problem if it's WIP but better add a license sooner than later, so contributions (and liability) are covered.
Here is some guidance: https://choosealicense.com/
